src/CommandingOfficers/Ave.java

```diff
@@ -536,6 +554,14 @@ protected void perform(MapMaster gameMap)
       // Normal CO-power boost.
       myCommander.addCOModifier(damageMod);
 
+      // Freeze enemies around each of Ave's units or buildings.
+      for( Unit victim : findVictims(gameMap) )
+        victim.isStunned = true;
```

Comment:
Do we need a `StunUnitEvent` or similar so we can move this into `getEvents()`? Maybe out of scope for this PR.

Reply:
Yes, and yes.
Same for refreshing units.